debian/control

```diff
@@ -7,7 +7,9 @@ Standards-Version: 3.9.4
 
 Package: google-cloud-ops-agent
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends},
+         ${misc:Depends},
+         rsyslog | system-log-daemon
```

Comment:
Explicit dependencies generally appear first, don't they? @quentinmit could confirm.

Reply:
I think you're right. Fixed in [43aedd1](https://github.com/GoogleCloudPlatform/ops-agent/pull/1696/commits/43aedd190a82f3218406bb9b6b7e408659fce333)